### PR TITLE
[T6A1][W11-C4]

### PR DESCRIPTION
**Pull request by yamgent**
~~Work in Progress~~ See comment below

### CODE REVIEW

**Comment by yamgent** [comment]
Ready for review

File: doc/UserGuide.md [hunk 1]
```diff
@@ -94,6 +104,21 @@ Examples:
...
+## Edits phone number for a person : `editphone`
+Edits the phone number for the specified person.<br>
+Format: `editphone INDEX PHONE`
+
+> Edits the person's phone number at the specified `INDEX`.
+  The index refers to the index number shown in the most recent listing.
+  
+Examples: 
+* `list`<br>
+  `editphone 2 98765432`<br>
```
Comment (okkhoy) [inline doc/UserGuide.md]: think of a way where you can provide one edit command, and edit the fields based on the input; which do you think would be more useful

File: src/seedu/addressbook/commands/ChangeThemeCommand.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.ui.Gui;
+
+public class ChangeThemeCommand extends Command {
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/ChangeThemeCommand.java]: this class could use some header comments!

File: src/seedu/addressbook/commands/EditPhoneCommand.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    @Override
+    public CommandResult execute() {
+        try {
+            final ReadOnlyPerson target = getTargetPerson();
+            
+            Phone newPhone = new Phone(newPhoneNumber, target.getPhone().isPrivate());
+            Person targetWithNewPhone = new Person(target.getName(), newPhone, target.getEmail(),
+                    target.getAddress(), target.getTags());
+
+            addressBook.removePerson(target);
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/EditPhoneCommand.java]: think of better ways to do the edit than remove and add!
what happens to the index of such edited person now?

File: src/seedu/addressbook/commands/ListTagCommand.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.commands;
+
+import java.util.List;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.Tag;
+
+public class ListTagCommand extends Command{
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/ListTagCommand.java]: again, header comments?

File: test/java/seedu/addressbook/logic/LogicTest.java [hunk 5]
```diff
@@ -277,6 +291,46 @@ public void execute_tryToViewMissingPerson_errorMessage() throws Exception {
     }
 
     @Test
+    public void execute_editPhone_invalidArgsFormat() throws Exception {
```
Comment (okkhoy) [inline test/java/seedu/addressbook/logic/LogicTest.java]: :100:  for writing quite exhaustive tests!

**Comment by okkhoy** [comment]
@yamgent @Cherc @DesperadoFlames 
Some comments added. Please ack & close the PR after reading comments.

**Comment by yamgent** [comment]
Thank you!

**Pull request closed without merging**